lib/Test/Nginx/Socket.pm

```diff
@@ -2372,6 +2396,14 @@ to run the specified number of duplicate requests for each test block. When it i
 
 Default to 1.
 
+=head2 stop_after_request
+
+By default, the nginx is not still running after the request. The error log is incompletely, missing logs after nginx stoped.
```

Comment:
"nginx is not still running" => "nginx is no longer running"
"The error log is incompletely" => "The error log is incomplete"
"stoped" => "stopped" (but I think "stops" is better here)
"after the request" => "after the first request"